setup.py

```diff
@@ -124,10 +124,7 @@ def run(self):
             rmtree(self.source_plugin_dir)
 
         try:
-            if self.plugins == "endpoints":
-                subprocess.check_call(build_plugins_command[platform.system()], shell=True)
-            else:
-                raise OSError("No such rule exists")
+            subprocess.check_call(build_plugins_command[platform.system()], shell=True)
```

Comment:
Any reason for changing this? IIRC this was done for the sagemaker plugin

Reply:
Must be a mistake. Fixed it.